settings/installer/ospd.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+installer:
+    type: ospd
+    short_type: ospd
+    user:
```

Comment:
user, undercloud, defaults,  and firewall keys can be removed from this file. These files are generated by Infrared according to the spec arguments